lib/block_account.go

```diff
@@ -42,6 +43,8 @@ func (b *BlockAccount) String() string {
 	return string(sebakcommon.MustJSONMarshal(b))
 }
 
+var BlockAccountObserver = observable.New()
```

Comment:
IHMO, In general, **package level vars** as using  **Package-global objects** can have surprising side effects.  Anyway, This var can be in another place. :)

Reply:
I don't like this usage also. I am totally agreed what you worry about.
However, the object should be accessed from everywhere.
So, I will just move it to another separated package.
Is it enough? or is there any alternative solution? @anarcher

Reply:
@kfangw  I understand why it has to exist in pkg-global var. It makes sense :-)